src/components/admin/previewComponent/inquiresPreview/InquiresPreview.styled.ts

```
@@ -0,0 +1,69 @@
+export const Title = styled.p`
+  font-size: 13px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+export const StateArea = styled.div`
+  display: flex;
+`;
+
+export const Date = styled.p`
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)